src/main/java/com/documaster/rms/noark/ws/noarkentities/MappeBase.java

```diff
@@ -215,6 +219,46 @@ public void setMappetype(Mappetype mappetype) {
 		return link(NOEKKELORD_LINK, noekkelord);
 	}
 
+	public LinkAction<TEntity> addKryssReferanseTilMappe(String... mappeIds) {
```

Comment:
Shouldn't `addKryssReferanseTilMappe` be changed to `addKryssreferanseTilMappe` as kryssreferanse is one word in all other places?

Applies to all added methods.

Reply:
Yes. Fixed.